packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -782,18 +745,6 @@ sdk.server('https://eu.api.example.com/v14');`)
       });
     });
 
-    // Run through and convert every schema we need to use into TS types.
-    await Promise.all(
```

Comment:
This promise was the source of a lot of the operation blocking work that would often lock up your terminal when generating an SDK.

Reply:
this was bugging me a lot, so glad you were able to fix this 🥳